Rakefile

```diff
@@ -230,6 +244,7 @@ task :ci do
   # Main library
   declare '✅ 2.1 / ✅ 2.2 / ✅ 2.3 / ✅ 2.4 / ✅ 2.5 / ✅ 2.6 / ✅ 2.7 / ✅ 3.0 / ✅ 3.1 / ✅ jruby' => 'bundle exec rake spec:main'
   declare '✅ 2.1 / ✅ 2.2 / ✅ 2.3 / ✅ 2.4 / ✅ 2.5 / ✅ 2.6 / ✅ 2.7 / ✅ 3.0 / ✅ 3.1 / ✅ jruby' => 'bundle exec appraisal core-old rake spec:main'
+  declare '✅ 2.1 / ✅ 2.2 / ✅ 2.3 / ✅ 2.4 / ✅ 2.5 / ✅ 2.6 / ✅ 2.7 / ✅ 3.0 / ✅ 3.1 / ✅ jruby' => 'bundle exec rake spec:appsec:main'
```

Comment:
I was wondering what these lines were about?

Reply:
[`declare` is method we defined](https://github.com/DataDog/dd-trace-rb/blob/8c7eb4ff9fa4564dd37d185d01aea8b84b64214d/Rakefile#L200), that pretty much just runs the command (in this case 'bundle exec rake spec:appsec:main'), but only for the versions of Ruby with ✅.

You will see a lot of ❌ for JRuby, for example, as we know some configurations don't support it.

It's our colorful, conditional CI matrix.